nomos-da/full-replication/src/lib.rs

```diff
@@ -142,45 +127,45 @@ pub struct CertificateVerificationParameters {
 }
 
 impl certificate::Certificate for Certificate {
-    type VerificationParameters = CertificateVerificationParameters;
-    type Signature = [u8; 32];
     type Id = [u8; 32];
-
-    fn signers(&self) -> Vec<bool> {
-        self.attestations.iter().map(|_| true).collect()
-    }
+    type Signature = [u8; 32];
+    type VerificationParameters = CertificateVerificationParameters;
 
     fn signature(&self) -> Self::Signature {
-        let signatures: Vec<u8> = self
-            .attestations
-            .iter()
-            .map(|attestation| attestation.attestation_signature())
-            .flatten()
-            .collect();
+        let mut signatures = Vec::new();
+        for attestation in &self.attestations {
+            signatures.extend_from_slice(attestation.signature());
+        }
+        signatures.sort();
         hash(signatures)
```

Comment:
The other way around. Otherwise we sort the final bytes, which doesn't make too much sense. Probably my commant was missleading as I asked to sort the signatures when I wanted to say attestations.

```suggestion
        let mut attestations = self.attestations.clone();
        attestations.sort();
        let mut signatures = Vec::new();
        for attestation in &self.attestations {
            signatures.extend_from_slice(attestation.signature());
        }
        hash(signatures)
```